x-pack/server/lib/constants.js

```diff
@@ -6,3 +6,16 @@
 
 export const XPACK_DEFAULT_ADMIN_EMAIL_UI_SETTING = 'xPack:defaultAdminEmail';
 export const XPACK_INFO_API_DEFAULT_POLL_FREQUENCY_IN_MILLIS = 30001; // 30 seconds
+export const RANKED_LICENSE_TYPES = [
+  'basic',
+  'standard',
+  'gold',
+  'platinum',
+  'trial',
+];
+export const LICENSE_STATUS = {
```

Comment:
If we expect these to be consumed on the client, should they be made available in a `common` directory, instead of `server`?